docs/src/_getting-started/part4.md

```diff
@@ -0,0 +1,194 @@
+---> 100%
+12:31:03  1 of 1 OK created table model analytics.authors ................................ [SELECT 2 in 0.59s]
+12:31:03
+12:31:03  Finished running 1 table model in 1.56s.
+12:31:03
+12:31:03  Completed successfully
+12:31:04
+12:31:04  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
+#
+```
+
+</div>
+
+You can check the data inside the database. There should now be a table inside the `analytics` schema called `authors` populated with data.
+
+## Run the complete pipeline
+
+To check that everything works together as a pipeline, we clean out once more and run the whole E(t)LT pipeline. Drop the tap_github.commits and the analytics.authors tables, clear the state using `meltano state clear dev:tap-github-to-target-postgres` and run the final pipeline alltogether:
+
+<div class="termy">
+
+```console
+$ meltano run tap-github hide-github-mails target-postgres dbt-postgres:run
+[warning  ] No state was found, complete import.
+ [info     ] INFO Starting sync of repository: sbalnojan/meltano-example-el [...]
+[info     ] INFO METRIC: {"type": "timer", "metric": "http_request_duration",[...]
+[info     ] INFO METRIC: {"type": "counter", "metric": "record_count", "value": 21 [...]
+[info     ] time=2022-09-22 12:42:57 name=target_postgres level=INFO message=Table '"commits"' [...]
+[...]
+---> 100%
+[info     ] Incremental state has been updated at 2022-09-22 12:42:58.260520.
+[info     ] Block run completed.           block_type=ExtractLoadBlocks err=None set_number=0 success=True
+[info     ] 12:43:19  Running with dbt=1.1.2 cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:20  Found 1 model, [...]
+[info     ] 12:43:20                       cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:20  Concurrency: 2 threads (target='dev') cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:20                       cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:20  1 of 1 START table model analytics.authors ..................................... [RUN] [...]
+---> 100%
+[info     ] 12:43:21  1 of 1 OK created table model analytics.authors .........[...]
+[info     ] 12:43:21                       cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:21  Finished running 1 table model in 1.34s. cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:21                       cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:21  Completed successfully cmd_type=command name=dbt-postgres stdio=stderr
+info     ] 12:43:21                       cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] 12:43:21  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1 cmd_type=command name=dbt-postgres stdio=stderr
+[info     ] Block run completed.           block_type=InvokerCommand err=None set_number=1 success=True
+```
+
+</div>
+
+There we have it, a complete E(t)LT pipeline.
```

Comment:
```suggestion
There we have it, a complete E(t)LT pipeline! Congratulations!
```

Thoughts on having a fun picture of Melty or something like that to celebrate?

Also, you can have them run `meltano dragon` to generate a dragon and celebrate that way 😄 Just thinking of a way to make it a bit more playful.

Reply:
Inline :melty-bouncy: maybe?